source/renderer/app/containers/staking/StakePoolsListPage.js

```diff
@@ -24,15 +25,25 @@ export default class StakePoolsListPage extends Component<Props> {
 
   render() {
     const { uiDialogs, staking, app, profile } = this.props.stores;
-    const { currentTheme } = profile;
+    const { currentTheme, currentLocale, environment } = profile;
     const { stakePools, delegatingStakePools } = staking;
+    const { network, rawNetwork } = environment;
+    const getPledgeAddressUrl = (pledge: string) =>
```

Comment:
@daniloprates here we need `pledgeAddress` as a prop.

Reply:
✅